project/Http4sPlugin.scala

```diff
@@ -291,13 +291,13 @@ object Http4sPlugin extends AutoPlugin {
     // error-prone merge conflicts in the dependencies below.
     val asyncHttpClient = "2.12.3"
     val blaze = "0.15.2"
-    val bobcats = "0.1-8c1dbba"
     val boopickle = "1.4.0"
     val caseInsensitive = "1.1.4"
     val cats = "2.6.1"
     val catsEffect = "2.5.3"
     val catsParse = "0.3.4"
     val circe = "0.14.1"
+    val crypto = "0.1-019ed6f"
```

Comment:
It doesn't really matter in this particular case because this is the only repo that should use it, but  it freaks me out to see a hash release in production code.  Can we give this a proper monotonic number?

Reply:
Absolutely. I was going to wait until we are happy with everything here before cutting a release but I can do that now.

Reply:
Done.